model/commitqueue/commit_queue.go

```diff
@@ -26,6 +26,7 @@ func (m *Module) UnmarshalBSON(in []byte) error { return mgobson.Unmarshal(in, m
 
 type CommitQueueItem struct {
 	Issue   string   `bson:"issue"`
+	Version string   `bson:"version,omitempty"`
```

Comment:
I think that's true, but I'm not sure it's a problem since this is only used to get the link when there is a version assigned to the item (https://github.com/evergreen-ci/evergreen/pull/2455/files/239dd7fa5106d4b97c4275f6f94ef42e645ed510#diff-7f9dabe9369339d415247be60a5d1b45R228)

Reply:
I believe the patch is also only created for the first item on the queue, but Jonathan can verify.

Reply:
what @ablack12 is saying is true of a PR commit queue while what @bsamek is saying is true of a CLI commit queue. 
There's no reason they couldn't be made consistent, but that's how they are now.

Reply:
I think that means that for the PR commit queue, the example is good, but for the CLI commit queue, it should be to /patch, not to /version.

Reply:
For the CLI piece I'm using existing code that cases on whether to use /patch or /version.

https://github.com/evergreen-ci/evergreen/blob/master/operations/patch_util.go#L325